extensions/wikia/ImageReview/ImageStateUpdater.class.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+class ImageStateUpdater extends WikiaModel {
+
+	/**
+	 * update image state
+	 * @param array $images
+	 * @param int $userId
+	 */
+	public function update( array $images, int $userId ) {
+		wfProfileIn( __METHOD__ );
+
+		$deletionList = [];
+		$statsInsert = [];
+
+		$sqlWhere = [
+			ImageReviewStates::APPROVED => [],
+			ImageReviewStates::REJECTED => [],
+			ImageReviewStates::QUESTIONABLE => [],
+		];
+
+		foreach ( $images as $image ) {
+			$sqlWhere[ $image['state'] ][] = "( wiki_id = $image[wikiId] AND page_id = $image[pageId]) ";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "state" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)